source/_integrations/nordpool.markdown

```diff
@@ -39,58 +39,92 @@ Currency:
+### Diagnostic sensors
 
-- Daily average
+| Sensor                    | Type              | Description                                                                       |
+| ------------------------- | ----------------- | --------------------------------------------------------------------------------- |
+| Currency                  | [Currency]        | The configured currency.                                                          |
+| Exchange rate             | Integer           | The exchange rate between the configure currency and Euro's.                      |
+| Last updated              | Datetime          | The time when the market prices were last updated.                                |
 
-The daily average sensor is not enabled by default.
+## Examples
 
-### Diagnostic sensors
+A template sensor to add VAT and fixed cost is useful to get the actual energy cost in the energy dashboard. 
+
+### UI Template
 
-- Last updated - indicates when the market price was last updated.
-- Currency - The selected currency.
-- Exchange rate - EUR is the base currency so will show the exchange rate used on the market place.
+Create a helper using the UI.
+1. Go to `Settings`, `Devices & Services` and choose the `Helpers` tab at the top.
+2. Select `Create helper` at the bottom right.
+3. Choose `Template` and `Template a sensor`.
+4. Enter the fields as shown below.
 
-The `Exchange rate` sensor is not enabled by default.
+The template below takes the current price attributes, adds 0.1293 EUR as fixed costs and adds 21% VAT.
 
-## Example
+<p class='img'>
+  <img src='/images/integrations/nordpool/nordpool_create_template.png' alt='Screenshot: Create template sensor'>
+</p>
 
-A simple template sensor to add VAT and a fixed cost from an `input_number` entity
+### YAML Template
+
+A template sensor to add VAT and a fixed cost from an helper entity `input_number.add_fixed_cost`.
 
 {% raw %}
 
 ```yaml
 template:
   - sensor:
-      - name: "Full SE3 current price"
-        unit_of_measurement: "SEK/kWh"
+      - name: "Nordpool"
+        unit_of_measurement: "EUR/kWh"
         state_class: measurement
         state: >
-          {% set cost = states('sensor.nord pool_se3_current_price') | float(0) %}
+          # create a variable with the current price
+          {% set cost = states('sensor.nord_pool_nl_current_price') | float(0) %}
+          # create a variable with the additional fixed cost
           {% set add_cost = states('input_number.add_fixed_cost') | float(0) %}
-          # Add fixed cost to the spot price and add VAT (25%)
+          # Add cost and additional fixed cost. Add VAT (25%) by multiplying with 1.25 and round to 2 digits: 
           {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
 ```
 
 {% endraw %}
+
+### Energy Dashboard
+
+To use the Nordpool integration in the Energy dashboard, use the `Use an entity with current price` option when configuring grid consumption and production.
+
+<p class='img'>
```

Comment:
I think we should use markdown to add images https://www.markdownguide.org/basic-syntax/#images

Reply:
I think it's ok. I've seen html tags being used for images in other places.

Reply:
In that case. If you're happy then approve and merge it 👍